src/xrGame/RegistryFuncs.h

```diff
@@ -6,5 +6,5 @@ void WriteRegistry_StrValue(LPCSTR rKeyName, const char* value);
 u32 const ReadRegistry_BinaryValue(LPCSTR rKeyName, u8* buffer_dest, u32 const buffer_size);
 void WriteRegistry_BinaryValue(LPCSTR rKeyName, u8 const* buffer_src, u32 const buffer_size);
 
-void ReadRegistry_DWValue(LPCSTR rKeyName, DWORD& value);
-void WriteRegistry_DWValue(LPCSTR rKeyName, const DWORD& value);
+void ReadRegistry_DWValue(LPCSTR rKeyName, u32& value);
+void WriteRegistry_DWValue(LPCSTR rKeyName, const u32& value);
```

Comment:
Uses WinAPI, should use WinAPI types.
```suggestion
void ReadRegistry_DWValue(LPCSTR rKeyName, DWORD& value);
void WriteRegistry_DWValue(LPCSTR rKeyName, const DWORD& value);
```